app/views/layouts/application.html.haml

```diff
@@ -1,9 +1,9 @@
 !!!
-%html{lang: "en"}
+%html.h-full{lang: "en"}
   %head
     = render "layouts/components/head"
 
   %body
-    = yield
+    = render "shared/navbars"
```

Comment:
we should still have a yield? that's where the content will get rendered no?

Reply:
so the yield is in the navbars partial. It's done this way so the content renders in the correct place